plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSourceProvider.java

```diff
@@ -255,4 +279,21 @@ public static Optional<HiveColumnHandle> toHiveColumnHandle(DeltaLakeColumnHandl
                 throw new IllegalArgumentException("Unsupported column mapping: " + columnMapping);
         }
     }
+
+    private static ConnectorPageSource createPages(long totalRowCount)
```

Comment:
createFixPageSource? createStaticPageSource? createPages sounds weird to me in this context

Reply:
renamed to `generatePagesWithNoColumns`

Reply:
eventually renamed to `generatePages`